drivers/fpga/fpga_eos_s3.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2021 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <eoss3_dev.h>
+
```

Comment:
Maybe we should put a guarding `ifndef` here and in every header file? It would prevent errors when including a specific header file more than one time. For example, this header file uses such guardians: https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/serial/uart_stm32.h#L12

Reply:
You are right, I added `ifndef` to header file.

Reply:
Thanks, but I also meant that this change should be made in every header file beside `drivers/fpga/fpga_eos_s3.h`. For example, if I'm not mistaken, `include/drivers/fpga.h` is still missing it.

Reply:
My bad, I changed it.